### PR TITLE
Added additional entries for IPF

### PR DESCRIPTION
**Pull request by Rosa-234**
I've added staining conditions for Podoplanin, UCHL1, SMAD3, TAK1, SMAD2, TGFBR2, pSMAD1/5, VWF and pTAK1, in human lung FFPE.

### CODE REVIEW

**Comment by zivy** [comment]
Hello @Rosa-234,

Thanks for the continued support of the Knoweldge-Base. I made some corrections on top of your original commit addressing multiple issues. Please review the updated pull request to ensure that non of the corrections is wrong. Thanks.

**Comment by Rosa-234** [comment]
Hi @zivy,

Thank you for your revisions, it looks good to me!

Thank you,
Rosa

File: docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md
Comment (radtkea) [inline docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md:1]: Looks good but I would delete the "k" from IgG1k. We do not specify the light chain (kappa or lambda).

File: .github/data_validation_configs/reagent_resources.json
Comment (radtkea) [inline .github/data_validation_configs/reagent_resources.json:1]: This is the same issue that I found. This should be changed to IgG1 not IgG1k.

File: docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md
Comment (zivy) [reply docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md:1]: Deleted.

File: .github/data_validation_configs/reagent_resources.json
Comment (zivy) [reply .github/data_validation_configs/reagent_resources.json:1]: The addition of IgG1k was undone, so the `reagent_resources.json` did not change and is no longer part of this PR.

**Comment by zivy** [comment]
Hello @radtkea, 
Please re-review and approve the PR if all of the issues were resolved correctly. Thanks.

File: docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md
Comment (radtkea) [reply docs/supporting_material/Podoplanin_Unconjugated/0000-0002-8728-1735.md:1]: Great, thank you!

**Review by radtkea — APPROVED** [review]
Looks good to me! Thank you so much:)

**Pull request merged**